test/interchaintest/interchainstaking_test.go

```diff
@@ -116,120 +116,121 @@ func TestInterchainStaking(t *testing.T) {
 	_ = quickUserAddr
 	_ = junoUserAddr
 
-	runSidecars(t, ctx, quicksilver, juno)
+	// runSidecars(t, ctx, quicksilver, juno)
 }
 
-func runSidecars(t *testing.T, ctx context.Context, quicksilver, juno *cosmos.CosmosChain) {
```

Comment:
if this is now handled by upstream logic, please feel free to remove.

Reply:
Maybe we should move these codes of `interchaintest` to another pr to make this pr more clearly
cc: @anhductn2001